scripts/typescript-references.js

```diff
@@ -0,0 +1,72 @@
+const cp = require('child_process')
```

Comment:
You said on discord that there are some packages now that do this too. Do you wish to keep this file here? Or use one npm package available out there?

Reply:
I don't see much interest in switching that now. No other new package on npm looks overly popular. Until we have problems I think it's ok to leave this as it is.